### PR TITLE
Fix/3163 linting errors on tracim frontend lib

### PR DESCRIPTION
**Pull request by raphj**
#3163

## Checkpoints

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

File: frontend_lib/src/component/ShareDownload/NewShareDownload.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export class NewShareDownload extends React.Component {
     }
   }
 
-  handleTogglePopoverMultipleEmails = () => {
+  popoverMultipleEmailsToggle = () => {
```
Comment (grignards) [inline frontend_lib/src/component/ShareDownload/NewShareDownload.jsx:22]: Wouldn't `toggleMultipleEmailsPopover` be clearer? (verb first).

**Comment by raphj** [comment]
For the tests: this PR does not change anything functionnal. This is just some linting, albeit a bit heavy. Just check that the main features work correctly.

File: frontend_lib/src/component/ShareDownload/NewShareDownload.jsx (hunk 1 above)
Comment (raphj) [reply frontend_lib/src/component/ShareDownload/NewShareDownload.jsx:22]: Indeed.

**Comment by gbonaspetti** [comment]
Tested on branch.

**Review by grignards — APPROVED** [review]

**Pull request merged**